src/main/kotlin/com/example/coreweb/commons/ResourceUtil.kt

```diff
@@ -8,11 +8,27 @@ import org.springframework.http.MediaType
 import org.springframework.http.ResponseEntity
 import java.io.File
 import java.io.FileInputStream
+import java.util.regex.Pattern
 
 class ResourceUtil private constructor() {
 
     companion object {
 
+        private const val phoneNumberRegexBD = "\\+?(88)?0?1[56789][0-9]{8}\\b" // this is for bangladeshi phone number
```

Comment:
Why is this validators on resource util? Please move it to `com.example.common.utils.Validator`
As per I can say, phone and email validators already exists on that class. So you can just use them.